eng/pipelines/templates/stages/archetype-sdk-management.yml

```diff
@@ -29,14 +29,21 @@ stages:
       vmImage: 'ubuntu-18.04'
 
     steps:
+      # Java 7 was removed from the hosted agents via https://github.com/actions/virtual-environments/issues/2446.
+      # Step copied from https://github.com/actions/virtual-environments/pull/2633/files#diff-38b50849f7a1c5a5f1b413356b0eb9517ed6aa31ba1c97afc1e0c07e62735f33
+      - script: |
+          sudo apt-get -y install zulu-7-azure-jdk=\*
+        condition: eq(variables['JavaVersion'], '1.7')
+
       - task: Maven@3
         displayName: 'Build'
         inputs:
           mavenPomFile: sdk/${{parameters.ServiceDirectory}}/pom.mgmt.xml
           goals: 'compile'
           options: '$(BuildOptions) $(ProfileFlag) "-DpackageOutputDirectory=$(Build.ArtifactStagingDirectory)" -DskipTests'
           mavenOptions: '$(MemoryOptions) $(LoggingOptions)'
-          javaHomeOption: 'JDKVersion'
+          javaHomeOption: path
+          jdkDirectory: '/usr/lib/jvm/zulu-7-azure-amd64'
```

Comment:
Won't this break for the java 1.8 builds?

Reply:
Ah, possible. Might need a condition as well? (https://docs.microsoft.com/en-us/azure/devops/pipelines/process/expressions?view=azure-devops#conditionally-set-a-task-input)